Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rocket-basicauth"
 description = "A high-level basic access authentication request guard for Rocket.rs"
-version = "2.1.1"
+version = "2.1.2"
```

Comment:
Could you bump the version to 3 thanks, will be a breaking change

Reply:
Oh and the readme too, almost missed that :)

Reply:
I updated the README and the version to 3.0.0